telegram/auth_status_test.go

```diff
@@ -48,13 +48,12 @@ func TestClient_AuthIfNecessary(t *testing.T) {
 	ctx := context.Background()
 
 	t.Run("Authorized", mockClient(func(a *rpcmock.Mock, client *Client) {
-		user := &tg.User{
+		testUser := &tg.User{
```

Comment:
Please don't increase review scope by renaming random variables, `u` or `user` is ok name by go naming conventions.

Reply:
Sorry about that! That was left from when I’ve had tg.User as return type and then tdakkota requested to change that to tg.AuthAuthorization (on Telegram), so looks like I forgot to revert some changes.